src/core/Pills/BasicPills/style.ts

```diff
@@ -0,0 +1,18 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    padding: theme.spacing(0.45, 1.5),
+    borderRadius: '0.1875rem',
+    fontSize: '0.625rem',
+    fontWeight: 500,
+    textTransform: 'none',
+    background: theme.palette.primary.light,
+    color: theme.palette.text.secondary,
+    '&:hover': {
+      background: theme.palette.primary.main,
+    },
```

Comment:
Hover state is not there in Plls.

selected: {} would be there which would be inferred from the card/div background selection

Reply:
I think you'd require a variant {default, selected}. Based on the props passed by the environment, you'd need to change the css property.